FakeNFT/Presentation/ProfileFlow/NFTCollection/NFTCollectionCell.swift

```diff
@@ -0,0 +1,176 @@
+        button.setImage(UIImage(named: "NotLiked"), for: .normal)
+        return button
+    }()
+
+    private lazy var infoStackView: UIStackView = {
+        let stackview = UIStackView()
+        stackview.translatesAutoresizingMaskIntoConstraints = false
+        stackview.distribution = .fill
+        stackview.spacing = 5
+        stackview.axis = .vertical
+        return stackview
+    }()
+
+    private lazy var nameLabel: UILabel = {
+        let label = UILabel()
+        label.font = .bodyBold
+        label.textColor = UIColor.NFTColor.black
+        label.translatesAutoresizingMaskIntoConstraints = false
+        return label
+    }()
+
+    private lazy var starsImageView: UIImageView = {
+        let imageView = UIImageView()
+        imageView.translatesAutoresizingMaskIntoConstraints = false
+        return imageView
+    }()
+
+    private lazy var authorStackView: UIStackView = {
+        let stack = UIStackView()
+        stack.translatesAutoresizingMaskIntoConstraints = false
+        stack.axis = .horizontal
+        stack.distribution = .fill
+        return stack
+    }()
+
+    private lazy var byWordLabel: UILabel = {
+        let label = UILabel()
+        label.font = .caption1
+        label.text = "от "
+        label.translatesAutoresizingMaskIntoConstraints = false
+        label.textColor = UIColor.NFTColor.black
+        return label
+    }()
+
+    private lazy var authorLabel: UILabel = {
+        let label = UILabel()
+        label.font = .caption2
+        label.translatesAutoresizingMaskIntoConstraints = false
+        label.textColor = UIColor.NFTColor.black
+        return label
+    }()
+
+    private lazy var priceStackView: UIStackView = {
+        let stackView = UIStackView()
+        stackView.axis = .vertical
+        stackView.distribution = .fill
+        stackView.translatesAutoresizingMaskIntoConstraints = false
+        return stackView
+    }()
+
+    private lazy var priceWordLabel: UILabel = {
+        let label = UILabel()
+        label.font = .caption2
+        label.text = "Цена"
+        label.translatesAutoresizingMaskIntoConstraints = false
+        return label
+    }()
+
+    private lazy var priceCashLabel: UILabel = {
+        let label = UILabel()
+        label.translatesAutoresizingMaskIntoConstraints = false
+        label.font = .bodyBold
+        return label
+    }()
+
+    // MARK: - Init
+    override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
+        super.init(style: style, reuseIdentifier: reuseIdentifier)
+        setupView()
+        activateConstraints()
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    // MARK: - Functions
+    func configCell(name: String, rating: Int, author: String, price: String, image: String, isLiked: Bool) {
+        nameLabel.text = name
+        authorLabel.text = author
+        priceCashLabel.text = "\(price) ETH"
```

Comment:
В макете цена NFT через точку с запятой, можно поправить через NumberFormatter или replacingOccurrences, например:
```suggestion
        priceCashLabel.text = "\(price) ETH".replacingOccurrences(of: ".", with: ",")
```